assessments/projects/in-toto/README.md

```diff
@@ -0,0 +1,98 @@
+# in-toto Security Assessment
+
+Completed: May 2019
+
+Security reviewers: Sarah Allen, Justin Cormack, Brandon Lum
+
+Project security lead: Santiago Torres Arias
+
+* Source code: [github.com/in-toto](https://github.com/in-toto)
+* Web site: https://in-toto.github.io/
+
+## Background
+[Supply chain compromises](https://github.com/in-toto/supply-chain-compromises)
+are a powerful attack vector. In cloud native deployments, if you control the
+supply chain you can potentially reconfigure anything in an insecure way, since
+everything is software-defined.
+
+In-toto ensures the integrity of the supply chain itself, specifically:
+1. What steps are to be carried in the supply chain
+2. Who can carry out each step
+3. How the artifacts between each step interconnect with each other
+
+_Maturity_
+* 1 public company case study: [Datadog Agent full
+pipeline](https://www.datadoghq.com/blog/engineering/secure-publication-of-datadog-agent-integrations-with-tuf-and-in-toto/)
+* multiple integrations: [Debian apt
+get](https://github.com/in-toto/apt-transport-in-toto) ([3 Debian
+contributors](https://salsa.debian.org/reproducible-builds/debian-rebuilder-setup/graphs/master)),
+[Jenkins plug-in](https://plugins.jenkins.io/in-toto) ([+1
+contributor](https://github.com/jenkinsci/in-toto-plugin/graphs/contributors)),
+[K8s admission controller](https://github.com/in-toto/in-toto-webhook) ([+1
+contributor](https://github.com/in-toto/in-toto-webhook/graphs/contributors))
+
+
+## Summary
+
+**Design**: straight-forward, adaptable to variations in supply chain pipeline
+and process. Non-prescriptive approach leaves opportunity for additional
+higher-level tooling.
+
+**Analysis**: focused approach to mitigating risks in supply chain which are
+underserved by other tech. Threats are well-understood and mitigated, including
+a reasonable degradation approach. Solid code processes; resiliency would
+benefit from additional participants.  
+
+All questions from reviewers were addressed in
+[self-assessment](self-assessment.md) with non-critical issues captured as
```

Comment:
This link is currently broken, should we include this file in the same PR as well?

Reply:
LOL.  filename had typo!  file merged in prior PR -- I just renamed to match this link.